### ISSUE TITLE
Quark incompatibility?

### ISSUE THREAD

**Issue by xannor** [PG85/OpenTerrainGenerator#15]
I am not sure if this is an isue with OTG or something else, but the only common thread I can determine is that some interaction between OTG and Quark will cause in world Quark items (blocks, models, etc) to either vanish or become a different block. This seems to occur on server restart, but possibly only when mod update occurs during, before, that restart.

Issue on quark repo: https://github.com/Vazkii/Quark/issues/416

Does OTG do anything to the anvil file or and special world load/block mapping mechanics that may have issue with Quark?

**Comment by PG85** [comment]
Hey Xannor, thanks for the report. OTG doesn't add any items or blocks and shouldn't have any effect on item/block id's, registry or save files. Of course there could be a bug somewhere, I'll need a bit more information though, does this bug ocurr with only OTG + Quark installed or are you using other mods as well?

**Comment by xannor** [comment]
I have a bunch of mods. On the issue I linked is another who is having a similar problem, and I listed the mods we had that were the same (and recently updated when I noticed the problem.) They also reported that they recently updated some mods and had a slew of world issues, though they didnt say what mods they updated.

**Comment by xannor** [comment]
At this point I cannot reproduce the issue and I know at least two updates to OTG have occurred since I originally noticed it. The update I did today should have reproduced the issue (I know the mods I updated added and removed blocks but I have no additional/further corruption. The best guess I have at this time is that at some point prior some sort of conflict occurred that caused the corruption, and it has since been resolved in some fashion. I will continue to do full backups each time I upgrade and if I get to a point where I can reproduce the issue I will open a new ticket.
